.travis.yml

```diff
@@ -12,11 +12,12 @@ services:
 install:
   - docker build -t cd-bot .
   - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then docker run -d -e BOT_TOKEN --name cd-bot cd-bot; fi'
-  - pip install -r requirements-ci.txt
+  - pip install poetry
```

Comment:
Why do we actually need to do this if Travis doesn't run the bot? Surely this just extends the build.